virttest/utils_misc.py

```diff
@@ -2056,6 +2056,34 @@ def compare_qemu_version(major, minor, update, is_rhev=True, params={}):
     return True
 
 
+def is_qemu_feature_supported(params, ignore_error=False):
+    """
+    Check whether the function is supported in this qemu-kvm version by comparing
```

Comment:
function-->feature?

Reply:
May I ask what the difference between 'function' and 'feature' is here?

Reply:
I've checked with chunfu, updated function name to 'is_qemu_function_supported' as discussed.